includes/orders/functions/actions.php

```diff
@@ -324,7 +324,7 @@ function edd_add_manual_order( $args = array() ) {
 							'object_id'   => $order_item_id,
 							'object_type' => 'order_item',
 							'type'        => 'tax_rate',
-							'total'       => $tax_rate->amount,
+							'total'       => number_format( $tax_rate->amount, 4 ),
```

Comment:
I think this may be incorrect, but I might also be mistaken.

I believe EDD always wants to store the exact amount to be as precise as possible, without truncating or rounding it. Truncation or rounding happens on display only, just for simplicity sake.

In the event the tax rate is `0.52759` in this case `number_format()` here would be saving an inaccurate value.

Reply:
That makes sense. I had added this back in because it was removed from the same code in the `edd_get_tax_rate` function, but based on what you're saying here, it should not have been added there, either.